src/api/metadata/metadata.spec.ts

```diff
@@ -13,14 +13,14 @@ const HTTP_STATUS_OK = 200
 const HTTP_STATUS_INTERNAL_ERROR = 500
 
 const DEFAULT_APP_DATA_DOC = {
-  version: '0.3.0',
+  version: '0.4.0',
```

Comment:
I'm assuming there s reason you're not just importing the version from package JSON?

Reply:
Good point, for the metadata will be different, but the main doc should always match the version and the package version. This could be a way of enforcing it

Reply:
The appData version does not follow the SDK package version.

It might, when it's exported to its own package